parser_library/src/workspaces/workspace.cpp

```diff
@@ -538,6 +538,10 @@ std::regex pathmask_to_regex(const std::string& input)
                 break;
 
             case '?':
+                path_started = true;
+                r.push_back('.');
```

Comment:
Yeah, I thought about it and then I decided that it might be a good idea to mimic the linux `find` with '-wholename' option which matches `/` as wells. What do you think?

Reply:
I'd never expect `ls -l /tmp?` or `ls -l /tmp?*` to list the contents of /tmp directory